airbyte-integrations/connectors/source-file-secure/Dockerfile

```diff
@@ -9,5 +9,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.26
+LABEL io.airbyte.version=0.2.27
```

Comment:
We should also be incrementing `FROM airbyte/source-file:0.2.26` to `0.2.27` to correspond to the `source-file` bump. However, for some reason I haven't been able to figure out immediately, using that version causes a SAT test to fail. The priority is to fix the build pipeline and not introduce a potential security regression so we should use the prior version instead of commenting out the acceptance test.